communication/mavsdkvehicleserver.cpp

```diff
@@ -42,7 +42,7 @@ MavsdkVehicleServer::MavsdkVehicleServer(QSharedPointer<VehicleState> vehicleSta
     mMissionRawServer.reset(new mavsdk::MissionRawServer(serverComponent));
 
     // Allow the vehicle to change to auto mode (manual is always allowed) and arm, disable takeoff (only rover support for now)
-    mActionServer->set_allowable_flight_modes({true, false, false});
+    mActionServer->set_allowable_flight_modes({true, true, false});
```

Comment:
should we put a comment what the true/false values are:
Auto/mission mode - > true
Guided mode -> true
Stabilize mode -> false

https://mavsdk.mavlink.io/main/en/cpp/api_reference/structmavsdk_1_1_action_server_1_1_allowable_flight_modes.html

Reply:
resolved: MAVSDK have comments in their code so you can see what the values are directly